website/docs/r/batch_job_definition.html.markdown

```diff
@@ -134,6 +134,40 @@ resource "aws_batch_job_definition" "test" {
 }
 ```
 
+### Job Definitionn of type EKS
```

Comment:
```suggestion
### Job Definition of type container using `ecs_properties`
```

I also think we should add examples using multicontainers definitions which is, to me, the key feature that was introduced.

https://docs.aws.amazon.com/batch/latest/userguide/multi-container-jobs-scenarios.html

Reply:
Agree, I added in https://github.com/hashicorp/terraform-provider-aws/pull/37871/commits/320085f325f146ae17fdc361c8a5e01de5ab8cee and https://github.com/hashicorp/terraform-provider-aws/pull/37871/commits/18ce938c07c3176c3a3dd20d0cde4e7c1937e745